### ISSUE TITLE
can't make rush work

### ISSUE THREAD

**Issue by abbood** [adamwiggins/rush#20]
originally i got the problem trying to run delayed jobs on [fedena](http://projectfedena.org/forum/4-project-fedena-souce-code-related/topics/766-exception-in-generate-reports) on a windows box.. but then even the most basic commands don't work:

``` shell
$ which rush
/c/Ruby187/bin/rush

$ rush
rush> echo "hello" 

Exception NoMethodError => undefine method 'echo' for localhost:Rush::Box
c:/Ruby187/lib/ruby/gems.18.gems/rush-0.6.7/lib/rush/shell.rb:37 in 'eval'
```

any ideas?

**Comment by Sija** [comment]
Yes. Rush is not made with Windows in mind and so most of the functionality might be missing or incomplete.
Secondly there’s no “echo” command.

**Comment by s-mage** [comment]
@abbood you're trying to run windows shell commands. Try "puts 'hello'", for example.
